src/core/geometry/qgsgeos.cpp

```diff
@@ -810,6 +810,33 @@ bool QgsGeos::intersects( const QgsAbstractGeometry *geom, QString *errorMsg ) c
   return relation( geom, RelationIntersects, errorMsg );
 }
 
+bool QgsGeos::intersects( double x, double y, QString *errorMsg ) const
```

Comment:
If we're adding this method then we need to take care that it's optimised for use in all situations -- can you model it off the QgsGeos::contains method please?

This will all eventually get cleaned up when we can safely bump the minimum geos version, so I'm not concerned about the complexity.